.circleci/test-multidev-merge-to-master.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+cd "$TARGET_REPO_WORKING_COPY"
+
+echo -e "\nMerging branch $TEST_BRANCH_NAME into master"
+
+git checkout master
+
+git merge -m 'Merge to master' $TEST_BRANCH_NAME
+
+git push origin master
+
+# Wait 30 seconds for the merge to register and the CI pipeline to start
+sleep 30
```

Comment:
Should `sleep 30` go at the end of this file, or should it be a step on its own? I don't think it's entirely intuitive to have it at the end of this file.